libs/types/src/election_package.ts

```diff
@@ -13,18 +13,17 @@ import { UiStringsPackage } from './ui_string_translations';
 export enum ElectionPackageFileName {
   APP_STRINGS = 'appStrings.json',
   AUDIO_CLIPS = 'audioClips.jsonl',
+  AUDIO_IDS = 'audioIds.json',
```

Comment:
Decided to rename `uiStringAudioIds.json` `audioIds.json` for consistency with `audioClips.jsonl`